frontend/src/scenes/insights/Insight.scss

```diff
@@ -314,6 +314,46 @@ $funnel_canvas_background: #fafafa;
     }
 }
 
+.dashboard-item {
+    .insight-empty-state {
+        &.match-container {
+            background-color: transparent;
+        }
+    }
+
+    &.purple {
+        .insight-empty-state {
+            &.match-container {
+                &.error {
+                    .illustration-main {
+                        color: $text_light;
+                    }
+
+                    h2 {
+                        color: $text_light;
+                    }
+                }
+            }
+        }
+    }
+
+    &:not(.purple) {
```

Comment:
Hm, not sure about giving _just_ purple special handling here - maybe all non-white should have this styling?

Reply:
I might pick up the storybook as described above then. Can check which colours do or don't work with white text then